### ISSUE TITLE
Error calling Detector API

### ISSUE THREAD

**Issue by rajvijay68** [therealexpertai/nlapi-python#11]
```python
document = client.detect(
		body={"document": {"text": text}}, 
		params={'language': language,'detector':'pii'})
```

```bash
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-24-329b1907077d> in <module>
      1 # cloud API
----> 2 pii_response = client.detect(
      3                 body={"document": {"text": text}},
      4 		params={'language': language,'detector':'pii'})

AttributeError: 'ExpertAiClient' object has no attribute 'detect'
```

**Comment by camato-es** [comment]
Thank you so much for your feedback , 
I'll keep you posted when the feature will be available.

**Comment by camato-es** [comment]
Hi,
we have released PII detection api in version 2.2.
now you can try to use it.
Regards
